code/game/dna/dna_modifier.dm

```diff
@@ -226,5 +226,4 @@
 				return
 		else
 	return
-
```

Comment:
Not sure why you removed the blank line here, but sure I guess?

Reply:
Probably have been a missclick but overall didnt change anything.